lte/gateway/c/core/oai/test/amf/test_amf_encode_decode.cpp

```diff
@@ -707,12 +716,14 @@ TEST(test_dlnastransport, test_dlnastransport) {
   msg.plain.amf.header.extended_protocol_discriminator =
       M5G_MOBILITY_MANAGEMENT_MESSAGES;
   msg.header.extended_protocol_discriminator = M5G_MOBILITY_MANAGEMENT_MESSAGES;
-  msg.plain.amf.header.message_type          = DLNASTRANSPORT;
+  msg.plain.amf.header.message_type =
+      static_cast<M5GMessageType>(M5GMessageType::DLNASTRANSPORT);
```

Comment:
What's the need to cast from "M5GMessageType" to "M5GMessageType"?

Reply:
addressed review comment, typecast is not needed